packages/features/eventtypes/components/AddMembersWithSwitch.tsx

```diff
@@ -1,11 +1,15 @@
 import type { ComponentProps, Dispatch, SetStateAction } from "react";
 import { useFormContext } from "react-hook-form";
+import { Controller } from "react-hook-form";
 import type { Options } from "react-select";
 
 import type { FormValues, Host, TeamMember } from "@calcom/features/eventtypes/lib/types";
 import { useLocale } from "@calcom/lib/hooks/useLocale";
-import { Label } from "@calcom/ui";
+import type { AttributesQueryValue } from "@calcom/lib/raqb/types";
+import { trpc } from "@calcom/trpc";
+import { Label, SettingsToggle } from "@calcom/ui";
 
+import { Segment } from "../../../../apps/web/components/Segment";
```

Comment:
I believe we should avoid importing apps code into package, always the other way around

Reply:
will make a follow up pr to move segment to a package and make it compatible with Platform